examples/.hashicorp_vault_token

```diff
@@ -0,0 +1 @@
+myroot
```

Comment:
This is the one for examples. I am adding this file here, so user does not have to create it while running the examples.